src/com/dis2/colorPicker/TestWheelSlider.java

```diff
@@ -5,21 +5,58 @@
  */
 package com.dis2.colorPicker;
 
+import com.dis2.progress.TestProgress;
+import java.awt.Color;
+import java.net.URL;
+import javax.swing.ImageIcon;
+import javax.swing.JFrame;
 import javax.swing.JPanel;
 
-
 /**
  *
  * @author David
  */
-public class TestWheelSlider extends JPanel {
+public class TestWheelSlider extends JFrame {
 
     public TestWheelSlider() {
-       
+        try {
+            WheelColor wc = new WheelColor(300, 300);
+            wc.setBackground(Color.black);
+
+            URL url = TestWheelSlider.class.getResource("/resources/bunny_brown.png");
+            wc.addCharacter(new Character(new Color(182, 123, 63), new ImageIcon(url).getImage()));
```

Comment:
There's a palette class for the colors, I think we can use that instead of creating a new Color object here.
Just to share our color palette across the app